torchft/manager_test.py

```diff
@@ -444,7 +444,7 @@ def test_pg_errored(self, client_mock: MagicMock) -> None:
         manager._pg.errored.return_value = injected_failure
 
         self.assertFalse(manager.should_commit())
-        self.assertEqual(manager._errored, injected_failure)
+        self.assertIsInstance(manager._errored, Exception)
```

Comment:
can we check .original_exception here instead? and for all the tests below including raise?

The type of error I think is pretty important for some of these tests to distinguish different code branches